ext/console/02_console.js

```diff
@@ -884,6 +900,7 @@
   }
 
   function inspectError(value) {
+    // TODO(@crowlKats): use circular system
```

Comment:
Can this TODO be addressed before landing?

Reply:
yes, will do